src/newusers.c

```diff
@@ -662,6 +662,13 @@ static void process_flags (int argc, char **argv)
 		case 's':
 			sflg = true;
                         bad_s = 0;
+
+			if (!crypt_method){
+				fprintf(stderr,
+						_("%s: Give crypt-method before number of rounds\n"),
```

Comment:
s/Give crypt-method/Provide '--crypt-method'

Reply:
You misunderstood this message. My suggestion was to replace `Give crypt-method` by `Provide '--crypt-method'` and keep the rest of the message, but you removed it. Let me write it in the final version:
```
%s: Provide '--crypt-method' before number of rounds\n"
```